CMake/cdat_modules_extra/setup_runtime.sh.in

```diff
@@ -10,10 +10,11 @@ function cleanup {
 # which can lead to errors.
 if [ -n "${UVCDAT_SETUP_PATH}" ] ; then
   if [ "${UVCDAT_SETUP_PATH}" = "${install_prefix}" ] ; then
+    echo "Nothing to do UVCDAT is already setup at "${UVCDAT_SETUP_PATH} 1>&2
```

Comment:
i definitely don't want the quotes in the output, and since we are doing an echo of strings the quotes are actually not necessary. What I have in there is right. But I really don't care and we want to keep @remram44 happy so I will push an update right away with the quotes around the var.